frontend/src/components/ui/avatar.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import * as AvatarPrimitive from "@radix-ui/react-avatar";
+
+// eslint-disable-next-line import/extensions
+import { cn } from "@/lib/utils";
+
+const Avatar = React.forwardRef<
+  React.ElementRef<typeof AvatarPrimitive.Root>,
+  React.ComponentPropsWithoutRef<typeof AvatarPrimitive.Root> & {
+    className?: string;
+  }
+>(({ className, ...props }, ref) => (
```

Comment:
In React components, it's a good practice to provide default values for props to ensure that the component behaves correctly even if some props are not provided. You can add some blanc default valuue like classNmae = ' ' for example